leaders/src/main/java/edu/cornell/Main.java

```diff
@@ -55,10 +59,20 @@ public static void main(String[] args) {
         } else {
             workerIds = Set.of(); // FIXME: Replace
         }
-        TestConsumer testConsumer = new PrintTestConsumer(workerIds);
-        try (KafkaConsumerRunner consumer =
-                new KafkaConsumerRunner(kafkaAddress, workerIds, testConsumer)) {
-            consumer.run();
+        Set<String> tests = Set.of("org.example.CalcTest"); // FIXME: Replace
+        TestConsumer testConsumer = TestConsumer.createTestConsumer(workerIds);
+        try (CloseableSet<Worker> workers = new CloseableSet<>();
+                WorkerRunner workerRunner = new WorkerRunner(workers);
+                KafkaConsumerRunner kafkaRunner =
+                    new KafkaConsumerRunner(kafkaAddress, workerIds, testConsumer)
+                ) {
+            workers.add(Worker.createWorker(url, branch, tests, kafkaAddress));
```

Comment:
This implementation consists of us creating an empty set of workers, then sticking it into the `WorkerRunner`, then adding workers to the worker set, then running the thing. This pattern absolutely sucks since it breaks the abstraction barrier. I'm going to change it for my own sanity